.gitignore

```diff
@@ -1,3 +1,4 @@
+.DS_Store
```

Comment:
Why not in your own global .gitignore?

Reply:
I agree. These files should be in global git ignore.

Reply:
Done